collection_requirements.rst

```diff
@@ -167,6 +167,9 @@ We should avoid FQCN / repository names:
 * which are unnecessary long: try to make it compact but clear
 * contain the same words / collocations in ``NAMESPACE`` and ``COLLECTION`` parts, for example ``my_system.my_system``
 
+If a collection is published on both Galaxy and Red Hat Automation Hub,
+it is desirable that the Galaxy and Automation Hub collection namespaces and names are identical if possible.
+Having two different names makes it impossible to mention the correct FQCNs in books, tutorials, documents, and in other sources since there is no single FQCN that always works, and it will cause confusion for users.
```

Comment:
```suggestion
"If your Collection is planned to be certified on Automation Hub, please consult with Red Hat Partner Engineering to ensure Collection naming compatibility between the community Collection on **Galaxy."**
```

Reply:
PTAL, I removed the double quotes and made "Collection" starting with a lowercase letter (as in other parts of the document)